scripts/metavar.py

```diff
@@ -1077,23 +1079,30 @@ def write_def(self, outfile, indent, wdict, allocatable=False,
                 raise CCPPError(errmsg.format(name))
             # end if
         # end if
+        optional = self.get_prop_value('optional')
         if protected and dummy:
             intent_str = 'intent(in)   '
         elif allocatable:
             if dimstr or polymorphic:
-                intent_str = 'allocatable  '
+                intent_str = 'allocatable         '
+                if target:
+                    intent_str = 'allocatable,'
+                    intent_str += '  target'
             else:
                 intent_str = ' '*13
             # end if
         elif intent is not None:
             alloval = self.get_prop_value('allocatable')
```

Comment:
Can this be True here given the `elif allocatable` above? If it can, then what if is also `optional`?

Reply:
I'm not sure of the purpose of alloval here?
If we are in this block of code, the variable is not an allocatable?